res/css/components/views/elements/_AppPermission.pcss

```diff
@@ -16,41 +16,31 @@ limitations under the License.
 */
 
 .mx_AppPermission {
-    > div {
-        margin-bottom: 12px;
-    }
-
-    h4 {
-        margin: 0;
-        padding: 0;
-    }
+    font-size: $font-12px;
+    width: 100%; /* make mx_AppPermission fill width of mx_AppTileBody so that scroll bar appears on the edge */
+    overflow-y: scroll;
 
-    .mx_AppPermission_smallText {
-        font-size: $font-12px;
-    }
+    .mx_AppPermission_content {
+        margin-block: auto; /* place at the center */
 
-    .mx_AppPermission_bolder {
-        font-weight: var(--font-semi-bold);
-    }
+        > div {
+            margin-block: 12px;
+        }
 
-    .mx_AppPermission_helpIcon {
-        margin-top: 1px;
-        margin-right: 2px;
```

Comment:
This `margin-right` property is no longer necessary. This has been used to add 2px space between the question mark and "with".